LoyaltyProgram_Intern/src/main/java/com/pgs/soft/domain/User.java

```diff
@@ -40,9 +40,15 @@
 	@JoinColumn(name = "user_profile_id")
 	private UserProfile userProfile = new UserProfile();
 	
+	private String uuid;
```

Comment:
Please rename field and all relevant methods to registrationToken - uuid is to generic